### PR TITLE
Laravel 11 integration and uses PHPUnit 10

### PR DESCRIPTION
**Pull request by crynobone**
This PR adds missing requirements for #7088

### CODE REVIEW

File: src/Features/SupportConsoleCommands/Tests/FileManipulationCommandParserUnitTest.php [hunk 1]
```diff
@@ -3,13 +3,12 @@
...
+use PHPUnit\Framework\Attributes\{Test, DataProvider};
 
 class FileManipulationCommandParserUnitTest extends \Tests\TestCase
 {
-    /**
-     * @test
-     * @dataProvider classPathProvider
-     */
+    #[Test]
+    #[DataProvider('classPathProvider')]
```
Comment (crynobone) [inline src/Features/SupportConsoleCommands/Tests/FileManipulationCommandParserUnitTest.php:11]: Using PHPUnit Annotation will trigger a deprecation warning in PHPUnit 11, it's best to migrate to PHPUnit Attributes

**Comment by calebporzio** [comment]
Thanks @crynobone!

**Comment by calebporzio** [comment]
Ooops, I thought this was to `main` - lmk if you want me to revert the merge.

**Comment by crynobone** [comment]
@calebporzio this is fine, it's part of `feat/l11` and that PR can be marked as ready now. @nunomaduro on holiday right now so it may take longer to wait for him.

**Pull request merged**